frontend/nc-games/src/api.js

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://backend-project-work.herokuapp.com/api", //source database url
+});
+
+export const getReviews = () => {
+    return api.get(`/reviews`).then((res) => {
+        return res.data;
+     });
+}
+
+export const getReview = (id) => {
+    return api.get(`/categories/${id}`).then((res) => {
```

Comment:
Heather was right - if you take a look at the endpoints you have access to on your BackEnd API - you need to go to /reviews/${id} to get an individual id. Categories/${id} doesn't exist (but it's not going to stop you achieving what is needed for this ticket.)